packages/neon-core/src/u/misc.ts

```diff
@@ -78,3 +79,39 @@ export function reverseHex(hex: string): string {
   }
   return out;
 }
+
+/**
+ * Calculates the byte size of any supported input following NEO's variable int format.
+ */
+export function getVarSize(value: any): number {
+  if (typeof value === "object" && value instanceof HexString) {
+    const size = value.byteLength;
+    return getVarSize(size) + size;
+  } else if (typeof value === "number") {
+    if (value < 0xfd) return 1;
+    else if (value <= 0xffff) return 3;
+    else return 5;
+  } else if (Array.isArray(value)) {
+    const array_length = value.length;
+    let size = 0;

```

Comment:
you can just:
```
let size = value.map(item => getVarSize(item)).reduce((prev, curr) => prev + curr, 0);
```

Reply:
good point, with a little edit it works as intended